apps/api-documenter/src/documenters/MarkdownDocumenter.ts

```diff
@@ -836,7 +836,7 @@ export class MarkdownDocumenter {
 
     const parametersTable: DocTable = new DocTable({
       configuration,
-      headerTitles: ['Parameter', 'Type', 'Description']
+      headerTitles: ['Parameter', 'Type', 'Optional', 'Description']
```

Comment:
@zelliott Rather than adding an additional column to the table, would it be better to indicate optional parameters similar to what we did for optional members of an interface?

Here is [an example](https://api.rushstack.io/pages/node-core-library.ifilesystemcopyfilebaseoptions/):
![image](https://user-images.githubusercontent.com/4673363/160724438-939aaa32-817a-4d7f-9cc9-8a68b3c5febe.png)